Tests/TransferMethodConfigurationResultTests.swift

```diff
@@ -215,13 +216,13 @@ class TransferMethodConfigurationResultTests: XCTestCase {
         }
 
         // Assert Field branchId
-        let fieldsFilterByBranchId = fields.filter { (field) in  field.name == "branchId" }
+        let fieldsFilterByBranchId = fields.filter { $0.name == "branchId" }
         if let branchId = fieldsFilterByBranchId.first {
             XCTAssertEqual(branchId.name, "branchId", "Invalid name")
             XCTAssertEqual(branchId.category, "ACCOUNT", "Invalid category")
             XCTAssertEqual(branchId.dataType, "NUMBER", "Invalid dataType")
-            XCTAssertEqual(branchId.isRequired, true, "Should not be required")
-            XCTAssertEqual(branchId.isEditable, true, "Should not be required")
+            XCTAssertEqual(branchId.isRequired, false, "Should not be required")
```

Comment:
You could use ```XCAssertTrue```

Reply:
You shouldn't change this. It's not part of the ticket. Ticket is about isEditable and not isRequired

Reply:
ok